packages/edit-site/src/components/editor/index.js

```diff
@@ -47,6 +47,15 @@ import { GlobalStylesProvider } from '../global-styles/global-styles-provider';
 import useTitle from '../routes/use-title';
 
 const interfaceLabels = {
+	/* translators: accessibility text for the editor top bar landmark region. */
+	header: __( 'Editor top bar' ),
+	/* translators: accessibility text for the editor content landmark region. */
+	body: __( 'Editor content' ),
+	/* translators: accessibility text for the editor settings landmark region. */
+	sidebar: __( 'Editor settings' ),
+	/* translators: accessibility text for the editor footer landmark region. */
```

Comment:
I think we should also update the `actions` landmark region label. Currently, it says "Publish".

https://github.com/WordPress/gutenberg/blob/f2888ed55f5aeae667d1cacc7c3ec22740f8952c/packages/edit-post/src/components/layout/index.js#L60-L61

Reply:
I wasn't sure about this one. My thinking was that 'publish' isn't quite the right term in the site editor, but I didn't realise the default is already 'publish' 😄 . I thought it would be 'Actions'.

Thanks for reviewing.

Reply:
Good point!

Reply:
I'll keep it as 'Editor publish' for now as I don't have any better ideas.